src/rpc/net.cpp

```diff
@@ -777,6 +778,7 @@ static RPCHelpMan listbanned()
         rec.pushKV("banned_until", banEntry.nBanUntil);
         rec.pushKV("ban_created", banEntry.nCreateTime);
         rec.pushKV("ban_time", (banEntry.nBanUntil - banEntry.nCreateTime));
+        rec.pushKV("ban_time_remaining", (banEntry.nBanUntil - GetTime()));
```

Comment:
Maybe query the time once before the loop, and use it for every iteration? This is needlessly slow and inconsistent.

Reply:
whoops, brain fart, forgot we were in a loop

Reply:
addressed in 833bebdece0f6fac5fecf602e88ef7c114f2416c